src/web/client/extension.ts

```diff
@@ -380,6 +380,12 @@ export function createWebWorkerInstance(
                             WebExtensionContext.userCollaborationProvider.refresh();
                             WebExtensionContext.quickPickProvider.refresh();
                         }
+                        if (data.type === Constants.workerEventMessages.TELEMETRY) {
+                            WebExtensionContext.telemetry.sendInfoTelemetry(
```

Comment:
you would want to log info and error as two different events

Reply:
addressed!